.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - 1.10.x
+  - master
+
+env:
+  - DEP_VERSION="0.4.1"
+
+before_install:
+  # Install dep as per https://golang.github.io/dep/docs/FAQ.html#how-do-i-use-dep-in-ci
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
+  - chmod +x $GOPATH/bin/dep
+  # Install metalinter
+  - go get github.com/alecthomas/gometalinter
```

Comment:
Thoughts on having a make task to install the build tools?

Reply:
I did think about it, but the `dep` download is platform-specific and I also really don't want make tasks that muck about with things outside the project's immediate environment.